lib/core/terraform_config/identity.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module TerraformConfig
+  class Identity < Base
+    attr_reader :gvc, :name, :description, :tags
+
+    def initialize(gvc:, name:, description: nil, tags: nil)
```

Comment:
@borela @rafaelgomesxyz 
Should we add support for all optional arguments (`aws_access_policy`, `azure_access_policy`, `network_resource`, ...) that can be configured for `cpln_identity` resource in terraform?

Reply:
Not necessary for the initial PR on this topic.